README.md

```diff
@@ -1,63 +1,66 @@
 # OpenDoas: a portable version of OpenBSD's `doas` command
 
-`doas` is a minimal replacement for the venerable `sudo`. It was
+[`doas`](https://en.wikipedia.org/wiki/Doas) is a minimal replacement for the venerable `sudo`. It was
 initially [written by Ted Unangst](http://www.tedunangst.com/flak/post/doas)
 of the OpenBSD project to provide 95% of the features of `sudo` with a
 fraction of the codebase.
 
-## Building and installation discouragements
+## Building and Installation Warnings
 
 There are a few steps you have to carefully consider before building and installing
-opendoas:
+OpenDoas:
 
-* There are less eyes on random doas ports, just because sudo had a vulnerability
+* There are fewer eyes on random `doas` ports, just because `sudo` had a vulnerability
   does not mean random doas ports are more secure if they are not reviewed
-  or pam is configured incorrectly.
-* If you want to use pam; You have to [configure pam](#pam-configuration)
-  and failing to do so correctly might leave a big open door.
-* Use the configure script.
+  or [PAM](https://en.wikipedia.org/wiki/Pluggable_authentication_module) is configured incorrectly.
+  * If you want to use PAM; You have to [configure PAM](#pam-configuration)
+    and failing to do so correctly might leave a big open door.
+  
+* Use the `configure` script.
 * Use the default make target.
-* If you really want to install a setuid binary that depends on
-  pam being correctly configured, use the make install target
-  to install the software.
+  * If you really want to install a setuid binary that depends on
+    PAM being correctly configured, use the `make install` target
+    to install the software.
 
-## About the port
+*Caveat emptor:* a [build and installation](https://axebase.net/blog/2020/10/28/doas-das-kleine-sudo/) example for Debian/Ubuntu derivatives (German).
```

Comment:
I removed the previous build instructions and I prefer to not include them. I don't want to have to deal with support requests about pam. There are distribution packages available for opendoas and they should be used. If they are not available, building software with configure and make should be easy enough for users who really want to configure pam and build opendoas themselves.

Reply:
Fair enough, I'll rescind that and work on the squash commit @ericonr suggested!